### PR TITLE
first draft

### PR DESCRIPTION
**Pull request by thmsmlr**

### CODE REVIEW

**Comment by thmsmlr** [comment]
#### 📝Changed routes:


| [**…ll-that-was-unexpected**](https://thmsmlrcom-git-well-that-was-unexpected.thmsmlr.now.sh/blog/well-that-was-unexpected) | [**/**](https://thmsmlrcom-git-well-that-was-unexpected.thmsmlr.now.sh/) |
|:-:|:-:|
|<a href="https://thmsmlrcom-git-well-that-was-unexpected.thmsmlr.now.sh/blog/well-that-was-unexpected">Screenshot of /blog/well-that-was-unexpected</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/hBZ62TKqcZDuC8eC9z_tLv_GgF4_.png">(view full size)</a> |<a href="https://thmsmlrcom-git-well-that-was-unexpected.thmsmlr.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/FwoDYOYgGQQ3XjeuZrmB_4AZ5mk_.png">(view full size)</a> |







Commit 5acf92666c472b68586fd30762f9bfae5025fde1 (https://thmsmlrcom-git-well-that-was-unexpected.thmsmlr.now.sh).

File: pages/blog/well-that-was-unexpected.js [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+    </p>
+    <h3>Conclusion</h3>
+    <p>
+      Well it was a fun 15 minute. I really want to thank the folks at{' '}
+      <a href="https://vercel.com">Vercel</a> and <a href="https://last.fm">Last.FM</a> for bank
+      rolling the meme. You didn't ask to, but it was very generous that you didn't shut me down. I
+      appreciate that.
+    </p>
+    <p>
+      If you're an engineer and what the tldr to make something scale. Put it on Vercel, make the
```
Comment (AlecStrong) [inline pages/blog/well-that-was-unexpected.js:209]:
```suggestion
      If you're an engineer and want the tldr to make something scale. Put it on Vercel, make the
```

**Comment by AlecStrong** [comment]
good read ;)

**Comment by thmsmlr** [comment]
Thanks buddy :)

**Pull request merged**